datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/manifest.py

```diff
@@ -0,0 +1,196 @@
+    "mac_os": "Supported OS::Mac OS",
+    "windows": "Supported OS::Windows",
+}
+
+CATEGORIES_TO_CLASSIFIER_TAGS = {
+    "alerting": "Category::Alerting",
+    "autodiscovery": "Category::Autodiscovery",
+    "automation": "Category::Automation",
+    "aws": "Category::AWS",
+    "azure": "Category::Azure",
+    "caching": "Category::Caching",
+    "cloud": "Category::Cloud",
+    "collaboration": "Category::Collaboration",
+    "compliance": "Category::Compliance",
+    "configuration & deployment": "Category::Configuration Deployment",
+    "containers": "Category::Containers",
+    "cost management": "Category::Cost Management",
+    "data store": "Category::Data Store",
+    "developer tools": "Category::Developer Tools",
+    "event management": "Category::Event Management",
+    "exceptions": "Category::Exceptions",
+    "google cloud": "Category::Google Cloud",
+    "incidents": "Category::Incidents",
+    "iot": "Category::IOT",
+    "isp": "Category::ISP",
+    "issue tracking": "Category::Issue Tracking",
+    "languages": "Category::Languages",
+    "log collection": "Category::Log Collection",
+    "marketplace": "Category::Marketplace",
+    "messaging": "Category::Messaging",
+    "metrics": "Category::Metrics",
+    "monitoring": "Category::Monitoring",
+    "network": "Category::Network",
+    "notification": "Category::Notification",
+    "oracle": "Category::Oracle",
+    "orchestration": "Category::Orchestration",
+    "os & system": "Category::OS System",
+    "processing": "Category::Processing",
+    "profiling": "Category::Profiling",
+    "provisioning": "Category::Provisioning",
+    "security": "Category::Security",
+    "snmp": "Category::SNMP",
+    "source control": "Category::Source Control",
+    "testing": "Category::Testing",
+    "web": "Category::Web",
+}
+
+
+@click.group(context_settings=CONTEXT_SETTINGS, short_help='Manifest utilities')
+def manifest():
+    pass
+
+
+@manifest.command(context_settings=CONTEXT_SETTINGS, short_help='Migrate a manifest to a newer schema version')
+@click.argument('integration', required=True)
+@click.argument('to_version', required=True)
+@click.pass_context
+def migrate(ctx, integration, to_version):
+    """
+    Helper tool to ease the migration of a manifest to a newer version, auto-filling fields when possible
+
+    Inputs:
+
+    integration: The name of the integration folder to perform the migration on
+
+    to_version: The schema version to upgrade the manifest to
+    """
+    echo_info(f"Migrating {integration} manifest to {to_version}....", nl=True)
+
+    # Perform input validations
+    if integration and integration not in get_valid_integrations():
+        abort(f'    Unknown integration `{integration}`')
+
+    loaded_manifest = JSONDict(load_manifest(integration))
+    manifest_version = loaded_manifest.get_path("/manifest_version")
+
+    if to_version not in SUPPORTED_MANIFEST_VERSIONS:
+        abort(f"    Unknown to_version: `{to_version}`")
```

Comment:
```suggestion
        abort(f"    Unknown to_version: `{to_version}`.  Valid options are: {SUPPORTED_MANIFEST_VERSIONS}")
```
Or something similar so it's obvious you need to put in the whole version string.

Reply:
I went through and updated a few of the log lines to try and be a bit more actionable